### PR TITLE
feat(manifests): sync with deis-dev charts. Assumes secrets already exist from a helm install

### PR DESCRIPTION
**Pull request by helgi**

### CODE REVIEW

**Comment by arschles** [comment]
@helgi LGTM-ed, assuming it works in testing

File: rootfs/etc/confd/templates/builder [hunk 1]
```diff
@@ -186,7 +187,7 @@ puts-step "Launching app."
 puts-step "Launching... "
 URL="http://$DEIS_WORKFLOW_SERVICE_HOST:$DEIS_WORKFLOW_SERVICE_PORT/v2/hooks/build"
 DATA="$(generate-buildhook "$SHORT_SHA" "$USER" "$APP_NAME" "$APP_NAME" "$PROCFILE" "$USING_DOCKERFILE")"
-PUBLISH_RELEASE=$(echo "$DATA" | publish-release-controller -url=$URL -key={{ getv "/deis/controller/builderKey" }})
+PUBLISH_RELEASE=$(echo "$DATA" | publish-release-controller -url=$URL -key="$BUILDER_KEY"
```
Comment (slack) [inline rootfs/etc/confd/templates/builder]: https://imgs.xkcd.com/comics/lisp_cycles.png

**Comment by helgi** [comment]
```
git push deis master
Counting objects: 77, done.
Compressing objects: 100% (72/72), done.
Writing objects: 100% (77/77), 17.43 KiB | 0 bytes/s, done.
Total 77 (delta 33), reused 0 (delta 0)
-----> Starting build
-----> Go app detected
-----> Checking Godeps/Godeps.json file.
-----> Installing go1.4.2... done
-----> Running: godep go install -tags heroku ./...
-----> Discovering process types
       Procfile declares types -> web
-----> Compiled slug size is 1.7M
-----> Build complete.
-----> Launching app.
-----> Launching...
       done, jicama-uptowner:v1 deployed to Deis

       Use `deis open` to view this application in your browser.

       To learn more, use `deis help` or visit http://deis.io

To ssh://git@deis.10.3.0.161.xip.io:2222/jicama-uptowner.git
 * [new branch]      master -> master
```

**Comment by helgi** [comment]
```
curl jicama-uptowner.10.3.0.161.xip.io
<html>
<head><title>404 Not Found</title></head>
<body bgcolor="white">
<center><h1>404 Not Found</h1></center>
<hr><center>nginx/1.9.6</center>
</body>
</html>
```

**Comment by helgi** [comment]
```
DEBUG create jicama-uptowner_v2.web, img jicama-uptowner:git-3e6fdde8, params {'version': 'v2', 'cpu': {}, 'tags': {}, 'aname': 'jicama-uptowner', 'healthcheck': {'timeout': 1, 'delay': 10, 'port': 8080, 'path': '/'}, 'num': 1, 'app_type': 'web', 'memory': {}, 'envs': {}}, cmd "start web"
DEBUG a bytes-like object is required, not 'str'
ERROR jicama-uptowner: jicama-uptowner_v2.web (scale): failed to get ReplicationController "jicama-uptowner-v2-web" in Namespace "jicama-uptowner": 404 Not Found
```

Workflow bug or it's outdated due to being `beta` tag... should have been fixed at this point tho. I'll have a look but it is unrelated

**Comment by helgi** [comment]
Works now with master of workflow

```
To ssh://git@deis.10.3.0.161.xip.io:2222/madcap-ricochet.git
 * [new branch]      master -> master
core@micro-kube ~/example-go $ curl madcap-ricochet.10.3.0.161.xip.io
curl: (6) Couldn't resolve host 'madcap-ricochet.10.3.0.161.xip.io'
core@micro-kube ~/example-go $ curl madcap-ricochet.10.3.0.161.xip.io
curl: (6) Couldn't resolve host 'madcap-ricochet.10.3.0.161.xip.io'
core@micro-kube ~/example-go $ curl madcap-ricochet.10.3.0.161.xip.io
curl: (6) Couldn't resolve host 'madcap-ricochet.10.3.0.161.xip.io'
core@micro-kube ~/example-go $ curl madcap-ricochet.10.3.0.161.xip.io
curl: (6) Couldn't resolve host 'madcap-ricochet.10.3.0.161.xip.io'
core@micro-kube ~/example-go $ curl madcap-ricochet.10.3.0.161.xip.io
Powered by Deis
Release v2 on madcap-ricochet-v2-web-d771p
```

For some reason the hostname wouldn't resolve for 3 minutes. Maybe router was acting up or slow at picking up changes

**Pull request merged**